### DISCUSSION TITLE
Update proposal: <podcast:chapters>

### DISCUSSION THREAD

**Discussion by keunes** [Podcastindex-org/podcast-namespace#473] [Clarification Needed]
As discussed in this thread: https://mastodon.social/@keunes/108957034896398550

Some podcasts (like PC2.0) have a placeholder chapter in case no chapters are available (yet), even though a chapter file is already linked in the feed. This may happen, for example, if a podcast(er) relies on the community to provide chapters, and they may come a few days after the episode. However, such placeholder chapter may be confusing for the end-users, who might think (as I did) that their app doesn't have cached the latest chapter information, or that there was an error parsing the chapters as there is only one and usually there are more.

Therefore I suggest we update the [podcast:chapters specs](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#chapters) such that:
* If the chapter file is absent or empty, even when linked from the feed, this must be expected and handled gracefully by apps. In this case, apps may display no chapter user interface or display information that chapters are not available yet.
* If the chapter file contains no chapters but human-readable info (in a format that should be defined, I suppose), apps may display this text (or ignore it and stick to the scenario above). (E.g. in AntennaPod there is a chapters button, which in this situation could have an info icon instead of previous/next icons; the usual modal could display the text instead of the chapters list.) This possibility can be leveraged by podcasters to advertise the fact that they do have chapters (even though they're not available yet) and explain when chapters are to be expected and why they're not available yet.

Cc: @daveajones

**Comment by theDanielJLewis** [comment]
I think this is good and you should consider the scenario when the episode metadata file contains other data but _not_ chapters. (It's a file for more than only chapters, so it's not only a chapters file.)

**Comment by samsethi** [comment]
I am trying to find out what was actually proposed as a change that I can actually implement.

**Comment by geeknews** [comment]
This is similar to the transcript tag; while a transcript is being created automatically at Blubrry, we place a message in the transcript file that says transcript coming soon. That is usually 6-7 minutes later. 

The reason we do this is we have no way of re-publishing the feed on PowerPress sites when the real file is delivered so we write a dummy file there that is later replaced on our server with the same name of the file containing the pending text in the feed. 

My suggestion is that you have some common text we place in the file so that apps will know that its placeholder

So I suggest the word be "placeholder" when apps see that they then ignore the chapter file until it has properly formatted xml.
